### DISCUSSION TITLE
The theme editor tab should be pinned during edit style or image.

### DISCUSSION THREAD

**Discussion by mindnect** [godotengine/godot-proposals#6706] [Editor]
Currently, clicking on a stylebox or image closes the theme editor tab.

To get back to the theme tab, I have to go back and find the theme file in the file tab or inspector, click on it again, and then open the theme tab again.

It should keep the Theme tab open while modifying the colors or margins etc of a style box, so that we can comfortably make changes and see the results live. 

It's a simple change, but it will greatly improve the devopment process.
If there is a way to pin the theme tab and I'm not aware of it, please let me know. Thanks.

**Comment by mindnect** [comment]
I noticed that editing the theme in the inspector tab only kept the theme editor tab open.
This seems to be a 4.x bug, it works fine in 3.5.x. I'll report it as a bug.

**Comment by hsandt** [comment]
As you noticed yourself when you opened an issue, there was already an issue about this. I'm just linking it here: https://github.com/godotengine/godot/issues/72271
